Multisig/UI/Transactions/TransactionDetails/AdvancedTransactionDetailsView.swift

```diff
@@ -39,3 +39,42 @@ struct AdvancedTransactionDetailsView: View {
         }
     }
 }
+
+struct AdvancedTransactionDetailsViewV2: View {
```

Comment:
Do we still need the old AdvancedTransactionDetailsView?

Reply:
it will be cleaned up later - see or description